app/assets/javascripts/discourse/app/lib/link-hashtags.js

```diff
@@ -15,21 +14,19 @@ export function linkSeenHashtags($elem) {
 
   const slugs = [...$hashtags.map((_, hashtag) => hashtag.innerText.substr(1))];
 
-  schedule("afterRender", () => {
-    $hashtags.each((index, hashtag) => {
-      let slug = slugs[index];
-      const hasTagSuffix = slug.endsWith(TAG_HASHTAG_POSTFIX);
-      if (hasTagSuffix) {
-        slug = slug.substr(0, slug.length - TAG_HASHTAG_POSTFIX.length);
-      }
-
-      const lowerSlug = slug.toLowerCase();
-      if (categoryHashtags[lowerSlug] && !hasTagSuffix) {
-        replaceSpan($(hashtag), slug, categoryHashtags[lowerSlug]);
-      } else if (tagHashtags[lowerSlug]) {
-        replaceSpan($(hashtag), slug, tagHashtags[lowerSlug]);
-      }
-    });
+  $hashtags.each((index, hashtag) => {
```

Comment:
Why was this removed from the `afterRender` callback? I guess it should depend on the site setting?

Reply:
I do not think this is necessary because it is called from inside a `afterRender` block already. 

```
d-editor.js / _updatePreview()
-> calls schedule("afterRender", ...)
-> calls composer-editor.js / previewUpdated
-> calls linkSeenHashtags
```